### ISSUE TITLE
Unable to open React Devtools

### ISSUE THREAD

**Issue by LegendEffects** [expo/expo#26640]
### Summary

When trying to open React Devtools from the CLI in a fresh project I get the following error:
```
/Users/evanbarker/Documents/my-app/node_modules/@expo/cli/build/src/start/interface/interactiveActions.js:189
        const url = this.devServerManager.getDefaultDevServer().getReactDevToolsUrl();
                                          ^

TypeError: Cannot read properties of undefined (reading 'getDefaultDevServer')
    at Object.openReactDevToolsAsync [as action] (/Users/evanbarker/Documents/my-app/node_modules/@expo/cli/build/src/start/interface/interactiveActions.js:189:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v20.5.1
```

In my existing project I get this error which is slightly different:
```
/Users/evanbarker/Documents/Work/app-v2-experiments/node_modules/flow-parser/flow_parser.js:818
throw a}function
^

TypeError: Cannot read properties of undefined (reading 'getDefaultDevServer')
    at Object.openReactDevToolsAsync [as action] (/Users/evanbarker/Documents/Work/app-v2-experiments/node_modules/@expo/cli/build/src/start/interface/interactiveActions.js:189:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v20.5.1
```

### What platform(s) does this occur on?

iOS

### SDK Version

50

### Environment

  expo-env-info 1.2.0 environment info:
    System:
      OS: macOS 14.3
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 20.5.1 - /opt/homebrew/bin/node
      Yarn: 1.22.19 - /opt/homebrew/bin/yarn
      npm: 9.8.0 - /opt/homebrew/bin/npm
      Watchman: 2023.08.14.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.12.1 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 23.2, iOS 17.2, macOS 14.2, tvOS 17.2, visionOS 1.0, watchOS 10.2
      Android SDK:
        API Levels: 26, 30, 31, 33
        Build Tools: 29.0.2, 29.0.3, 30.0.2, 30.0.3, 31.0.0, 33.0.0, 34.0.0
        System Images: android-26 | Google APIs ARM 64 v8a, android-27 | Google APIs ARM 64 v8a, android-28 | Google APIs ARM 64 v8a, android-31 | Google APIs ARM 64 v8a, android-34 | Google APIs ARM 64 v8a
    IDEs:
      Xcode: 15.2/15C500b - /usr/bin/xcodebuild
    npmPackages:
      expo: ^50.0.0 => 50.0.2 
      react: 18.2.0 => 18.2.0 
      react-dom: 18.2.0 => 18.2.0 
      react-native: 0.73.2 => 0.73.2 
      react-native-web: ~0.19.6 => 0.19.9 
    Expo Workflow: bare

### Minimal reproducible example

Reproducible in the default app template.

`npx create-expo-app my-app`

`npm run ios`

Navigate to launch React devtools (shift + m), see error.

**Comment by byCedric** [comment]
@brentvatne this is resolved by https://github.com/expo/expo/pull/26550, but needs to be published I think.

**Comment by brentvatne** [comment]
this is resolved in expo@50.0.3. run `npx expo install --fix` to update.

thanks for the report!

**Comment by LegendEffects** [comment]
Thanks both, can confirm that the fix has worked.
